pkg/storage/storagebackend/factory/tls_test.go

```diff
@@ -67,7 +68,7 @@ func TestTLSConnection(t *testing.T) {
 	}
 }
 
-func configureTLSCerts(t *testing.T) (certFile, keyFile, caFile string) {
+func configureTLSCerts(t *testing.T) (tmpDir, certFile, keyFile, caFile string) {
 	baseDir := os.TempDir()
```

Comment:
I think this code have done what you care about. `if err != nil` the `tempDir `will be empty.
```
 if err != nil {
		t.Fatal(err)
	}
```